### ISSUE TITLE
Double Content-Type header 

### ISSUE THREAD

**Issue by natanrolnik** [luckymarmot/Paw-RubyNetHTTPCodeGenerator#6]
Hi!

I created a request that works fine via Paw. However, using the Ruby Net::Http generator, it creates a duplicate `Content-Type` header:

```ruby
    # Create Request
    req =  Net::HTTP::Post.new(uri)
    # Add headers
    req.add_field "X-My-Custom-Header", "somevalueformyheader"
    # Add headers
    req.add_field "Content-Type", "application/json; charset=utf-8"
    # Set header and body
    req.add_field "Content-Type", "application/json"
    req.body = body
```

When running it from my rails app, the request failed. I needed to remove the line `req.add_field "Content-Type", "application/json; charset=utf-8"` in order to make it work.

**Comment by JonathanMontane** [comment]
I'll look into it! Thanks for the report

**Comment by JonathanMontane** [comment]
fixed in PR #7

**Comment by natanrolnik** [comment]
That was fast! Thanks @JonathanMontane
